### PR TITLE
[stable/grafana] fix init image tag

### PR DESCRIPTION
**Pull request by kimxogus**
Signed-off-by: Taehyun Kim <kgyoo8232@gmail.com>

#### What this PR does / why we need it:
`initChownData.image.tag: 1.30` will be interpreted as number and becomes `1.3` which results `Init:ImagePullBackOff` as `busybox:1.3` doesn't exists

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

**Comment by kimxogus** [comment]
changed tag to `latest` as many other charts use `latest` image and we can use those pre-downloaded images.

File: stable/grafana/values.yaml [hunk 1]
```diff
@@ -158,7 +158,7 @@ initChownData:
   ##
   image:
     repository: busybox
-    tag: 1.30
+    tag: "latest"
```
Comment (zanhsieh) [inline stable/grafana/values.yaml]: I think busybox:1.30 should be ok.
`$ docker pull busybox:1.30
1.30: Pulling from library/busybox
697743189b6d: Pull complete
Digest: sha256:061ca9704a714ee3e8b80523ec720c64f6209ad3f97c0ff7cb9ec7d19f15149f
Status: Downloaded newer image for busybox:1.30
`

File: stable/grafana/values.yaml [hunk 2]
```diff
@@ -26,7 +26,7 @@ livenessProbe:
 
 image:
   repository: grafana/grafana
-  tag: 6.0.2
+  tag: "6.0.2"
```
Comment (maorfr) [inline stable/grafana/values.yaml]: quotes are not required here. 6.0.2 is interpreted as a string.

File: stable/grafana/values.yaml (hunk 1 above)
Comment (maorfr) [reply stable/grafana/values.yaml]: i agree that 1.30 is ok. i'm not a fan of latest. "1.30" would probably do the trick.

Comment (kimxogus) [reply stable/grafana/values.yaml]: set `1.30`

File: stable/grafana/values.yaml (hunk 2 above)
Comment (kimxogus) [reply stable/grafana/values.yaml]: I know that, but just wanted to make sure. Should I revert this?

**Review by zanhsieh — APPROVED** [review]

File: stable/grafana/values.yaml (hunk 2 above)
Comment (maorfr) [reply stable/grafana/values.yaml]: yes, please

Comment (kimxogus) [reply stable/grafana/values.yaml]: reverted :)

**Comment by maorfr** [comment]
/lgtm

@kimxogus thanks for contributing, this is an important fix!

**Pull request merged**